core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatter3DChart/Scatter3dChartOpDesc.scala

```diff
@@ -0,0 +1,121 @@
+package edu.uci.ics.texera.workflow.operators.visualization.scatter3DChart
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "title": "string"
+  }
+}
+""")
+class Scatter3dChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(value = "title", required = true)
```

Comment:
You still need to add the default value here. What you added is the property description.

Reply:
OK, I misunderstood what you meant before, and now I think it's right?
<img width="771" alt="image" src="https://github.com/Texera/texera/assets/62498853/2bb710ab-f06b-402f-bc50-2c96ff650e4a">

Reply:
Yes.